tests/test_fellow.py

```diff
@@ -5,17 +5,29 @@
 from mock import patch, Mock
 
 
-class Test(unittest.TestCase):
+class TestFellow(unittest.TestCase):
 
     def setUp(self):
         self.fellow = Fellow()
+        self.fellow.id_no = 1234
 
     def test_id_value(self):
-        self.assertNotEqual(self.fellow.id_no, 0, msg="Id number not sumitted.")
-
-	@patch('app.storage')
-	def test_fellow_has_room_allocation(self,test_storage):
-		test_storage.people_info.retun_value = {12345678 : "Marvin kangethe"}
-		self.assertTrue(self.fellow.is_allocated(12345678), msg = "You already have room allocated.")
-  
- 
\ No newline at end of file
+        self.assertNotEqual(self.fellow.id_no, 0)
+
+    def test_id_type(self):    
+    	self.assertEqual(type(self.fellow.id_no), int)
+
+    def test_id_size(self):	
```

Comment:
Check line 12. Its an obvious test saying id=1234 and checking if the id length is 4